src/PrestaShopBundle/Bridge/AdminController/FrameworkBridgeControllerListTrait.php

```diff
@@ -28,91 +28,64 @@
 
 namespace PrestaShopBundle\Bridge\AdminController;
 
-use PrestaShopBundle\Bridge\AdminController\Action\ActionInterface;
-use PrestaShopBundle\Bridge\Exception\NotAllowedActionTypeForListException;
-use PrestaShopBundle\Bridge\Helper\Listing\Action\ListBulkAction;
-use PrestaShopBundle\Bridge\Helper\Listing\Action\ListHeaderToolbarAction;
-use PrestaShopBundle\Bridge\Helper\Listing\Action\ListRowAction;
-use PrestaShopBundle\Bridge\Helper\Listing\Field\FieldInterface;
-use PrestaShopBundle\Bridge\Helper\Listing\FiltersProcessor;
 use PrestaShopBundle\Bridge\Helper\Listing\HelperListConfiguration;
+use Symfony\Component\HttpFoundation\Request;
 
 /**
  * Contains the principal methods you need to horizontally migrate a controller which has a list.
  */
 trait FrameworkBridgeControllerListTrait
 {
+    /**
+     * @param string $identifierKey
+     * @param string $positionIdentifierKey
+     * @param string $defaultOrderBy
+     * @param string $postSubmitRoute
+     * @param bool $autoJoinLangTable
+     * @param bool $deleted
+     * @param bool $explicitSelect
+     * @param bool $useFoundRows
+     * @param string|null $listId
```

Comment:
Please describe all parameters

Reply:
Description is still missing here, but I guess you can copy what you added in the `HelperListConfiguration` class